### PR TITLE
change folder refs from preprod to prod

### PR DESCRIPTION
**Pull request by haraprasadj**
Link to Jira ticket if there is one:

### CODE REVIEW

**Review by xritter1 — APPROVED** [review]

File: healdata.org/dashboard/Public/notebooks/cdc_wonder_demo.html [hunk 1]
```diff
@@ -5460,7 +5460,7 @@
   content:"?"; }
 
 .bp3-icon-euro::before{
-  content:"�"; }
+  content:"�"; }
```
Comment (williamhaley) [inline healdata.org/dashboard/Public/notebooks/cdc_wonder_demo.html:5463]: Is this change intentional? I wonder if this change snuck in here somehow from a copy/paste or find/replace or an IDE auto-correcting something?

Comment (haraprasadj) [reply healdata.org/dashboard/Public/notebooks/cdc_wonder_demo.html:5463]: oh, thanks for the catch, let me revert that

**Review by williamhaley — APPROVED** [review]
